data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/node/GetPeers.java

```diff
@@ -88,10 +90,26 @@
                   eth2Peer.connectionInitiatedLocally() ? Direction.outbound : Direction.inbound)
           .build();
 
-  private static final SerializableTypeDefinition<List<Eth2Peer>> PEERS_RESPONSE_TYPE =
-      SerializableTypeDefinition.<List<Eth2Peer>>object()
+  private static final DeserializableTypeDefinition<UInt64> COUNT_TYPE =
+      DeserializableTypeDefinition.string(UInt64.class)
+          .formatter(UInt64::toString)
+          .parser(UInt64::valueOf)
+          .example("1")
+          .description("Total number of items")
+          .format("uint64")
+          .build();
+
+  private static final SerializableTypeDefinition<UInt64> PEERS_META_TYPE =
+      SerializableTypeDefinition.object(UInt64.class)
+          .name("Meta")
+          .withField("count", COUNT_TYPE, Function.identity())
```

Comment:
so `INTEGER_TYPE` will be used here instead of `COUNT_TYPE`, then `COUNT_TYPE` won't be needed (likely to need to remove a reference file when that happens)

Reply:
I've added description to `INTEGER_TYPE`, but example field is not easy to add to our API cleanly. Could we skip it?

Reply:
we can skip the example, and just basically use `INTEGER_TYPE.withDescription("...")` where `COUNT_TYPE` is in the withField fn call...